runtime/oak/src/lib.rs

```diff
@@ -1059,7 +1045,7 @@ impl pallet_valve::Config for Runtime {
 	type WeightInfo = pallet_valve::weights::SubstrateWeight<Runtime>;
 	type ClosedCallFilter = ClosedCallFilter;
 	type AutomationTime = AutomationTime;
-	type AutomationPrice = AutomationPrice;
+	type AutomationPrice = AutomationTime;
```

Comment:
This is weird. Why not just add a dummy implementation to `()`?

Reply:
```
error[E0117]: only traits defined in the current crate can be implemented for arbitrary types
      --> /Users/justinzhou/Documents/OAK/ENG/OAK-blockchain/runtime/oak/src/lib.rs:1043:1
       |
  1043 | impl pallet_valve::Shutdown for () {
       | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^--
       | |                               |
       | |                               this is not defined in the current crate because tuples are always foreign
       | impl doesn't use only types from inside the current crate
       |
       = note: define and implement a trait or new type instead
```

it didn't like impl for (), so just did impl on a new struct called DummyPallet. Lemme know if this works!

Reply:
You need to define the `impl Shutdown for ()` in the valve pallet. Probably makes sense to do so right in `valve/src/traits.rs` file as a default implementation.

The rule is that either the trait being implemented or the struct on which its being implemented needs to be defined within the crate doing the implementation. Since neither `Shutdown` or `()` are defined in the runtime it can't be implemented here.

Reply:
ahh i gotcha, thanks!